<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1604</width>
    <height>1036</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>1381</width>
      <height>761</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 12pt &quot;Lucida Sans&quot;;
background-color: rgb(230, 237, 255);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1381</width>
       <height>723</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Milestone 2 - Page 1</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>961</width>
        <height>291</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Lucida Sans&quot;;
background-color: rgb(196, 205, 255);
</string>
      </property>
      <property name="title">
       <string>Select Location</string>
      </property>
      <widget class="QLabel" name="statelabel">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>State</string>
       </property>
      </widget>
      <widget class="QLabel" name="citylabel">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>30</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>City</string>
       </property>
      </widget>
      <widget class="QLabel" name="ziplabel">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>30</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Zipcode</string>
       </property>
      </widget>
      <widget class="QComboBox" name="stateList">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>191</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Lucida Sans&quot;;
background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QListWidget" name="cityList">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>70</y>
         <width>281</width>
         <height>191</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 9pt &quot;Lucida Sans&quot;;</string>
       </property>
      </widget>
      <widget class="QListWidget" name="zipList">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>70</y>
         <width>281</width>
         <height>192</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
font: 9pt &quot;Lucida Sans&quot;;
background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>1341</width>
        <height>381</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Lucida Sans&quot;;
background-color: rgb(196, 205, 255);</string>
      </property>
      <property name="title">
       <string>Search Business </string>
      </property>
      <widget class="QLineEdit" name="bnameLine">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>30</y>
         <width>321</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 9pt &quot;Lucida Sans&quot;;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="bzipLine">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>90</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(196, 205, 255);
font: 63 9pt &quot;Lucida Sans&quot;;</string>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>40</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Enter Business Name:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Zipcode:</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="businessTable">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>100</y>
         <width>1091</width>
         <height>261</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 9pt &quot;Lucida Sans&quot;;</string>
       </property>
      </widget>
      <zorder>label_2</zorder>
      <zorder>bnameLine</zorder>
      <zorder>bzipLine</zorder>
      <zorder>label</zorder>
      <zorder>businessTable</zorder>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>0</y>
        <width>351</width>
        <height>291</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Lucida Sans&quot;;
background-color: rgb(196, 205, 255);</string>
      </property>
      <property name="title">
       <string>Categories</string>
      </property>
      <widget class="QListWidget" name="categoriesList">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>291</width>
         <height>231</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 9pt &quot;Lucida Sans&quot;;</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1604</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
